flask/cli.py

```diff
@@ -21,6 +21,7 @@
 from functools import update_wrapper
 from operator import attrgetter
 from threading import Lock, Thread
+from werkzeug import __version__ as werkzeug_version
```

Comment:
What about importing werkzeug and referencing the version with `werkzeug.__version__`?
It is clear enough, and it is not as if it is being used anywhere else.